core/tests/karma.conf.ts

```diff
@@ -164,6 +164,10 @@ module.exports = function(config) {
               }
             }
           ]
+        },
+        {
+          test: /\.html$/,
+          loader: 'underscore-template-loader'
```

Comment:
Please check if we really need this, we might not need this for tests.

Reply:
I think it is required

Reply:
Do the tests work without it?

Reply:
No tests were failing before you suggested it the after adding it they passed.